Testscripts/Linux/LSVMBUS.sh

```diff
@@ -146,19 +147,31 @@ else
     expected_network_counter=$VCPU
 fi
 
-# the cpu count that attached to the SCSI driver is MIN(the number of vCPUs/4, 64).
-if [ "$VCPU" -gt 64 ];then
-    expected_scsi_counter=64
-else
-    expected_scsi_counter=$(bc <<<"scale=2;$VCPU/4")
-    if [ "$expected_scsi_counter" != "${1#*.00}" ]; then
-        # In this case we have a float number that needs to be rounded up.
-        # For example with 6 cores, scsi controller is spread on 2 cores.
-        expected_scsi_counter=$(bc <<<"("$expected_scsi_counter"+0.5)/1")
-    fi
-    # normalizing the number to integer
-    expected_scsi_counter=${expected_scsi_counter%.*}
-fi
+case $VM_Size in
+    # If Lv2, the cpu count that attached to the SCSI driver is MIN(the number of vCPUs, 64).
+    *Standard_L*s_v2*)
```

Comment:
I think we have been talking about this way before.  Is there any other way we can count the number of expected scsi devices? If there is no other way, we should post the official document URL in the comment here. So we can track the change.

Reply:
also '64' can be in the constant form.

Reply:
Hi @juhlee-microsoft ,
Thanks for your comments! The PS1 script here is not for counting CPU number. It's for passing the VM size to the shell script. I have tested this case in different VM sizes and found that the SCSI controller counting method in Lv2 series is different from other VM series(as I mentioned in #820).

Reply:
Interesting. So this code is for specific Ls_v2 vm size. Thanks for letting me know.